### PR TITLE
Translate to ruby from ... (ko)

### PR DESCRIPTION
**Pull request by marocchino**
You can found original text from:

https://github.com/ruby/www.ruby-lang.org/blob/master/en/documentation/ruby-from-other-languages/to-ruby-from-c-and-cpp/index.md
https://github.com/ruby/www.ruby-lang.org/blob/master/en/documentation/ruby-from-other-languages/to-ruby-from-perl/index.md
https://github.com/ruby/www.ruby-lang.org/blob/master/en/documentation/ruby-from-other-languages/to-ruby-from-php/index.md

related https://github.com/ruby/www.ruby-lang.org/issues/2818

### CODE REVIEW

File: ko/documentation/ruby-from-other-languages/to-ruby-from-c-and-cpp/index.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+* 대부분의 경우 모든 것이 표현식입니다(즉, `while` 문은 실제로 rvalue로
+  평가됩니다).
+
+### C++과 다른 점
+
+C과는 다르게, Ruby에서는...
+
+* 명시적인 참조가 없습니다. 즉, Ruby에서 모든 변수는 일부 개체에 대해 자동으로
+  역참조된 이름일 뿐입니다.
+* 개체는 강 타입이지만 *동적* 타입이 지정됩니다. 런타임은 *런타임에* 해당 메서드
```
Comment (riseshia) [inline ko/documentation/ruby-from-other-languages/to-ruby-from-c-and-cpp/index.md:104]:
```suggestion
* 개체는 강 타입이지만 *동적* 타입이 지정됩니다. 런타임은 *실행 시간에* 해당 메서드
```
(maybe)

File: ko/documentation/ruby-from-other-languages/to-ruby-from-c-and-cpp/index.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  호출이 실제로 작동하는지 확인합니다.
+* "constructor"는 클래스 이름 대신 `initialize`라고 합니다.
+* 모든 메서드는 항상 가상입니다.
+* "클래스"(정적) 변수 이름은 항상 `@@`로 시작합니다.(예: `@@total_widgets`)
+* 멤버 변수에 직접 액세스하지 않습니다. 공용 멤버 변수(Ruby에서 attributes로
+  부름)에 대한 모든 액세스는 메서드를 통해 이루어집니다.
+* `this`가 아니라 `self`입니다.
+* 일부 메서드는 '?' 또는 '!'로 끝납니다. 실제로 메소드 이름의 일부입니다.
+* 그 자체로는 다중 상속이 없습니다. Ruby에는 "mixins"가 있습니다(즉, 모듈의 모든
```
Comment (riseshia) [inline ko/documentation/ruby-from-other-languages/to-ruby-from-c-and-cpp/index.md:113]:
```suggestion
* 그 자체로는 다중 상속이 없습니다. Ruby에는 "믹스인"이 있습니다(즉, 모듈의 모든
```
(maybe)

File: ko/documentation/ruby-from-other-languages/to-ruby-from-perl/index.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+---
+layout: page
+title: "Perl에서 Ruby로"
+lang: ko
+---
+
+펄은 굉장합니다. Perl의 문서도 굉장합니다. Perl 커뮤니티도… 굉장합니다.
```
Comment (riseshia) [inline ko/documentation/ruby-from-other-languages/to-ruby-from-perl/index.md:7]:
```suggestion
Perl은 굉장합니다. Perl의 문서도 굉장합니다. Perl 커뮤니티도… 굉장합니다.
```

File: ko/documentation/ruby-from-other-languages/to-ruby-from-perl/index.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+처음부터 내장된 우아한 OO 기능을 갈망하는 Perlers에게는 Ruby가 적합할 수
```
Comment (riseshia) [inline ko/documentation/ruby-from-other-languages/to-ruby-from-perl/index.md:8]:
```suggestion
처음부터 내장된 우아한 XX 기능을 갈망하는 Perlers에게는 Ruby가 적합할 수
```
(maybe) better to use 'xx' for korean? 🤔

File: ko/documentation/ruby-from-other-languages/to-ruby-from-perl/index.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+* 괄호는 종종 생략할 수 있습니다.
+* 문자열은 기본적으로 동일하게 작동합니다.
+* Perl과 유사한 일반 구분 문자열 및 정규식 인용 구문이 있습니다.
+  `%q{this}`(작은따옴표) 또는 `%Q{this}`(큰따옴표) 및 `%w{단일따옴표 단어 목록의
+  경우}`처럼 보입니다. 원하는 경우 `%Q|can|` `%Q(use)` `%Q^other^` 구분 기호를
+  사용할 수 있습니다.
+* `"#{like} this"`처럼 보이지만 큰따옴표 변수 보간이 있습니다.(그리고 `#{}` 안에
+  원하는 Ruby 코드를 넣을 수 있습니다)
+* 셸 명령 확장은 `` `backticks` ``를 사용합니다.
+* 포함된 문서 도구가 있습니다(Ruby는 rdoc이라고 함).
```
Comment (riseshia) [inline ko/documentation/ruby-from-other-languages/to-ruby-from-perl/index.md:27]:
```suggestion
* 내장된 문서 도구가 있습니다(Ruby는 rdoc이라고 함).
```

File: ko/documentation/ruby-from-other-languages/to-ruby-from-perl/index.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+* `$` 및 `@`는 변수 이름의 첫 번째 문자로 사용되는 경우가 있지만 유형을 나타내는
+  것이 아니라 범위를 나타냅니다.(`$`는 전역, `@`는 객체 인스턴스, `@@`는 클래스
+  속성을 나타냅니다)
+* 배열 리터럴은 괄호 대신 괄호 안에 들어갑니다.
+* 배열의 하위 배열을 작성해도 하나의 큰 배열로 병합되지 않습니다. 대신 배열의
+  배열을 얻습니다.
+* `sub`가 아니라 `def`를 사용합니다.
+* 각 줄의 끝에는 세미콜론이 필요하지 않습니다. 여기에 더해 함수 정의, 클래스
+  정의 및 case 문과 같은 것을 `end` 키워드로 끝냅니다.
+* 개체는 강타입입니다. 유형 간 변환이 필요한 경우 `foo.to_i`, `foo.to_s` 등을
```
Comment (riseshia) [inline ko/documentation/ruby-from-other-languages/to-ruby-from-perl/index.md:45]:
```suggestion
* 개체는 강타입입니다. 타입 간 변환이 필요한 경우 `foo.to_i`, `foo.to_s` 등을
```

File: ko/documentation/ruby-from-other-languages/to-ruby-from-php/index.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+* Ruby는 PHP와 같이 동적으로 타이핑되므로 변수 선언에 대해 걱정할 필요가
+  없습니다.
+* 클래스가 있으며 PHP 5(`public`, `protected` 및 `private`)에서와 같이 클래스에
+  대한 액세스를 제어할 수 있습니다.
+* 일부 변수는 PHP와 같이 $로 시작합니다(전부가 아님).
+* 'eval'도 있습니다.
+* 문자열 보간을 사용할 수 있습니다. `"$foo is a $bar"` 대신
+  `"#{foo} is a #{bar}"`를 사용할 수 있습니다. PHP에서와 같이 작은따옴표
+  문자열에는 적용되지 않습니다.
+* heredocs가 있습니다.
```
Comment (riseshia) [inline ko/documentation/ruby-from-other-languages/to-ruby-from-php/index.md:23]:
```suggestion
* 히어독(heredoc)이 있습니다.
```
(maybe)

File: ko/documentation/ruby-from-other-languages/to-ruby-from-php/index.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+* Ruby에는 PHP 5와 같은 예외가 있습니다.
```
Comment (riseshia) [inline ko/documentation/ruby-from-other-languages/to-ruby-from-php/index.md:24]:
```suggestion
* Ruby에는 PHP 5처럼 예외가 있습니다.
```

File: ko/documentation/ruby-from-other-languages/to-ruby-from-php/index.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+* 상당히 큰 표준 라이브러리가 있습니다.
+* 배열 및 해시는 `array()`를 `{`, `}`로 교환하면 예상대로 작동합니다.
+  `array('a' => 'b')`는 `{'a' => 'b'}`가 됩니다.
+* `true` 및 `false`는 PHP에서와 같이 동작하지만 `null`은 `nil`이라고 합니다.
```
Comment (riseshia) [inline ko/documentation/ruby-from-other-languages/to-ruby-from-php/index.md:28]:
```suggestion
* `true` 및 `false`는 PHP처럼 동작하지만 `null`은 `nil`이라고 합니다.
```
(maybe)

File: ko/documentation/ruby-from-other-languages/to-ruby-from-php/index.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+  않고 문자열, 정수 등을 변환합니다.
+* 문자열, 숫자, 배열, 해시 등은 객체입니다. abs(-1)을 호출하는 대신
+  -1.abs입니다.
+* 메서드 호출에서 괄호는 생략 가능합니다. 단, 어떤 매개변수가 어떤 메서드 호출로
+  이동하는지 명확히 하는 경우는 예외입니다.
+* 표준 라이브러리 및 확장은 모듈 및 클래스로 구성됩니다.
+* 리플렉션은 객체 고유의 기능이므로 PHP 5에서와 같이 `Reflection` 클래스를
+  사용할 필요가 없습니다.
+* 변수는 참조입니다.
+* `abstract` 클래스나 `interfaces`가 없습니다.
```
Comment (riseshia) [inline ko/documentation/ruby-from-other-languages/to-ruby-from-php/index.md:44]:
```suggestion
* `abstract` 클래스나 `interface`가 없습니다.
```

File: ko/documentation/ruby-from-other-languages/to-ruby-from-perl/index.md (hunk 4 above)
Comment (marocchino) [reply ko/documentation/ruby-from-other-languages/to-ruby-from-perl/index.md:8]: I thought it stand for Object Oriented

File: ko/documentation/ruby-from-other-languages/to-ruby-from-c-and-cpp/index.md [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+예상할 수 있습니다. 동시에 Ruby 프로그램을 얼마나 빨리 시작하고 실행할 수
+있는지, 작성하는 데 몇 줄의 코드가 필요한지 빠르게 이해할 수 있습니다.
+Ruby는 C++ 보다 훨씬 간단합니다. Ruby는 여러분을 응석받이로 만들 것입니다.
+
+Ruby는 정적으로 타입이 결정되지 않고 동적으로 결정됩니다. 런타임은 실행 시간에
+가능한 한 많은 작업을 수행합니다. 예를 들어, Ruby 프로그램이 "링크"(즉, 로드 및
+사용)할 모듈이나 미리 호출할 메서드를 알 필요가 없습니다.
+
+다행스럽게도 Ruby와 C는 건강한 공생 관계를 가지고 있습니다. Ruby는 so
+"확장 모듈"을 지원합니다. 이들은 Ruby 프로그램에서 사용할 수 있지만(외부에서 볼
```
Comment (marocchino) [inline ko/documentation/ruby-from-other-languages/to-ruby-from-c-and-cpp/index.md:25]: so-called appeared near by do block statement. so, I unify sentence with that.
```suggestion
다행스럽게도 Ruby와 C는 건강한 공생 관계를 가지고 있습니다. Ruby는 소위 
"확장 모듈"을 지원합니다. 이들은 Ruby 프로그램에서 사용할 수 있지만(외부에서 볼
```

File: ko/documentation/ruby-from-other-languages/to-ruby-from-perl/index.md (hunk 4 above)
Comment (marocchino) [reply ko/documentation/ruby-from-other-languages/to-ruby-from-perl/index.md:8]: Japanese say
> 最初から洗練されたオブジェクト指向の機能が組み込まれた言語を求めているPerl使いにとって、 Rubyは答えになるかもしれません。

So, It is Object Oriented. maybe We can replace it like:
```suggestion
처음부터 내장된 우아한 객체 지향 기능을 갈망하는 Perlers에게는 Ruby가 적합할 수
```

Comment (riseshia) [reply ko/documentation/ruby-from-other-languages/to-ruby-from-perl/index.md]: oh 🙈 lgtm

**Review by riseshia — APPROVED** [review]
Thanks!!

**Pull request merged**